runtime/doc/autocmd.txt

```diff
@@ -787,6 +787,31 @@ ModeChanged			After changing the mode. The pattern is
 		:au ModeChanged [vV\x16]*:* let &l:rnu = mode() =~# '^[vV\x16]'
 		:au ModeChanged *:[vV\x16]* let &l:rnu = mode() =~# '^[vV\x16]'
 		:au WinEnter,WinLeave * let &l:rnu = mode() =~# '^[vV\x16]'
+Progress						*Progress*
+				After a progress message is created via
+				`nvim_echo`. The pattern is matched against
+				title of the message. The |event-data| contains:
+					msg_id: id of the message
+					content: text of the message
```

Comment:
```suggestion
					text: text of the message
```

Reply:
msg_show ui+event already uses content for this.  Should we give a different name for the autocmd event?